src/cmd/moby/run_vmware.go

```diff
@@ -151,6 +151,13 @@ func buildVMX(cpus int, mem int, diskPath string, prefix string) string {
 
 	if diskPath != "" {
 		returnString += fmt.Sprintf(vmxDisk, diskPath)
+	} else {
+		vmdkPath := prefix + ".vmdk"
+		if _, err := os.Stat(vmdkPath); os.IsNotExist(err) {
```

Comment:
The error will be reported from `vmrun` if it can't access the vmdk file (permissions etc.), we could also pre-test for that condition?